docs/js/features/connectivity/on-premise.md

```diff
@@ -0,0 +1,49 @@
+---
+title: On-Premise Connectivity in the SAP Cloud SDK for JavaScript
+sidebar_label: On-Premise Systems
+description: This article describes how the SDK request to On-Premise SAP S/4 HANA systems.
+keywords:
+- sap
+- cloud
+- sdk
+- On-Premise
+- connectivity
+- JavaScript
+- TypeScript
+---
+
+
+## Introduction ##
+
+This document has overlap to the [proxy handling](./proxy.md), [destination lookup](./destination.md) and [On-Premise connectivity from the BAS](../../guides/bas-external-system.md).
+The focus of this document is on the steps necessary to connect to an On-Premise system and how the SDK implements these steps.
+The crucial problem with On-Premise SAP S/4 HANA systems is, that they are not exposed to the internet.
+Typically, they are located within the companies internal network.
+
+Since this isolation would be in conjunction to modern requirements of data availability, SAP has introduced the [Cloud Connector](https://help.sap.com/viewer/cca91383641e40ffbe03bdc78f00f681/Cloud/en-US/e6c7616abb5710148cfcf3e75d96d596.html?q=cloud%20connector).
+The Cloud Connector works like a reverse proxy opening a connection between some white-listed service endpoints of the On-Premise system, and one Cloud Foundry account.
+Only applications running in the connected Cloud Foundry account can access the services. 
+Applications which are supposed to access On-Premise systems need a binding to the [connectivity service.](https://www.cloudfoundry.org/the-foundry/sap-cloud-platform-service-connectivity/)
+
+
+## Implementation Details ##
```

Comment:
This title is not very inspiring. But to be fair I don't have anything better in mind.